### ISSUE TITLE
Use System Font for chat bubbles

### ISSUE THREAD

**Issue by andi98** [peter-iakovlev/Telegram#15]
Hi!
Since iOS 9 the system font is San Fransico. In the chat text box this font is used but if I send a message and it appears in a chat bubble, another font is used. I think it would look better, if this would be the same (San Fransico) font.
Would be great :)

**Comment by milsirhc** [comment]
Hi Andi, 

Did you manage to build this app?

**Comment by andi98** [comment]
No, I didn't tried. I just use the "Telegram Messanger" from the App Store (https://itunes.apple.com/de/app/telegram-messenger/id686449807?mt=8) - isn't this the repo for this app?
